.github/workflows/deploy-prod.yml

```
@@ -149,6 +178,7 @@ jobs:
       image_tag: ${{ needs.resolve-release.outputs.release_tag }}
       commit_sha: ${{ needs.resolve-release.outputs.commit_sha }}
       deploy_actor: ${{ github.actor }}
+      commit_sha: ${{ needs.resolve-release.outputs.commit_sha }}
```

actionlint (1.7.12)

[error] key "commit_sha" is duplicated in "with" section. previously defined at line:179,col:7. note that this key is case insensitive

(syntax-check)